WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
 RSpec.configure do |config|
-  config.before(:suite) do
-    DatabaseCleaner.clean_with :truncation
-    TestDbManager.fill_tables
+  unless ENV['SKIP_PRETEST_SETUP']
```

Comment:
Accepting any value that is truthy (i.e. "smells" like a value that could be boolean `true`) is bad practice in general.
Try resorting to a method like `unless ENV['FOO_BAR'].present?` or `if ENV['FOO_BAR'].nil?` which have clearly defined behaviour.

Reply:
How about the latest push, which explicitly checks `== 'true'`? I agree with the concern re false positives, but am concerned that checking for presence leads to false positives where  `FOO_BAR=false` would resolve as true when checking `ENV['FOO_BAR'].present?`

Reply:
This allows one to toggle true/false on SKIP_PRETEST_SETUP, which is nice if running the full suite or a set of tests which does rely on the pre-populated test data

Reply:
why not have it as an optional part of EnvConfig and then you can do `SKIP_PRETEST_SETUP?`

Reply:
Oh yeah I like that a lot! Was trying to do it similar to `ENABLE_BULLET` but couldn't figure out how. Just done it in the latest push.